### PR TITLE
Remove city vouchers

### PR DESCRIPTION
**Pull request by lucasvanhalst**
Steps:
- before deploying remove the module from every service
- update app.yaml & cron.yaml
- run from solutions.common.migrations.remove_vouchers import remove_unused_models
- after deploying cleanup indexes

### CODE REVIEW

File: src/solutions/common/bizz/city_vouchers.py [hunk 1]
```diff
@@ -14,294 +14,49 @@
...
-    # 2.2/ It is a city voucher
-    data = json.loads(poke_information.tag[len(POKE_TAG_CITY_VOUCHER_QR):])
-    ancestor_key = SolutionCityVoucher.create_parent_key(data["app_id"])
-    sln_city_voucher = SolutionCityVoucher.get_by_id(data["voucher_id"], ancestor_key)
-    if not sln_city_voucher:
-        logging.debug("Could not find city voucher for data: %s", data)
-        raise Exception("Could not find city voucher")
-
-    sln_settings = get_solution_settings(service_user)
+    logging.debug('Unknown QR code scanned: %s.', url)
```
Comment (RubenMattan) [inline src/solutions/common/bizz/city_vouchers.py:55]: logging.debug('Unknown QR code scanned: %s. Loyalty device will create custom paper loyalty card.', url)

File: src/solutions/common/bizz/city_vouchers.py [hunk 2]
```diff
@@ -318,255 +73,10 @@ def solution_voucher_resolve(service_user, email, method, params, tag, service_i
...
-            'title': common_translate(sln_settings.main_language, 'Transaction successful'),
-            'content': common_translate(sln_settings.main_language, 'voucher_value_is',
-                                        currency=sln_settings.currency_symbol, value=round(value_left / 100.0, 2))
-        }
-        result = json.dumps(r_dict)
-        r.result = result if isinstance(result, unicode) else result.decode("utf8")
-    except:
-        logging.error("solutions.voucher.redeem.confirm exception occurred", exc_info=True)
-        r.error = common_translate(sln_settings.main_language, 'error-occured-unknown')
+        r.error = translate(sln_settings.main_language, 'error-occured-unknown')
```
Comment (RubenMattan) [inline src/solutions/common/bizz/city_vouchers.py:81]: return missing

File: src/static/js/solutions/sln-loyalty.js [hunk 3]
```diff
@@ -88,6 +86,7 @@ $(function() {
         });
     }
 
+    // TODO lucas: replace CITY_APP_ID with nothign and add it on the server instead
```
Comment (RubenMattan) [inline src/static/js/solutions/sln-loyalty.js:89]: todo found

File: src/solutions/common/restapi/loyalty.py [hunk 4]
```diff
@@ -782,23 +788,25 @@ def close_city_wide_lottery_info(key):
                      service_identity=ll_info.service_identity)
```
Comment (lucasvanhalst) [inline src/solutions/common/restapi/loyalty.py:788]: Did this ever work? property `ll_info.service_identity` doesn't exist...

Comment (RubenMattan) [reply src/solutions/common/restapi/loyalty.py:788]: hmm does is use None then?

File: src/solutions/common/migrations/remove_vouchers.py [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+#
+# @@license_version:1.5@@
+
+from google.appengine.ext import ndb, db
+
+from rogerthat.utils.models import delete_all_models_by_query
+from solutions.common.models import SolutionSettings
+
+
+class AssociationStatistic(ndb.Model):
```
Comment (RubenMattan) [inline src/solutions/common/migrations/remove_vouchers.py:24]: service stats cleanup is not removed yet i guess. So keep this model for now

**Pull request merged**